src/plugins/intel_cpu/src/ngraph_transformations/convert_fq_rnn_to_quantized_rnn.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+/*
+ * Description:
+ *     ConvertFqRnnToQuantizedRnn detects RNN / LSTM / GRU_RNN operations
+ *     with FQ operations on the inputs and forms a new TypeRelaxed operation
+ *     with quantization parameters as runtime parameters of the operation.
+ *     @todo add ascii graph examples
+ */
+
+namespace ov {
+namespace intel_cpu {
+
+class ConvertFqRnnToQuantizedRnn: public ngraph::pass::MatcherPass {
```

Comment:
Minor: can we use ov::pass::MatcherPass as a base class here?

Reply:
Sometimes I am also trying to use ov namespace and then replace it back with ngraph.
I propose to leave ngraph namespace since it is used in every other file
We better refactor it in all the files at once